tiledb/libtiledb.pyx

```diff
@@ -4308,36 +4308,31 @@ cdef class Array(object):
 
         """
         cdef Domain dom = self.schema.domain
-        dom_dims = [dom.dim(idx) for idx in range(dom.ndim)]
-        dom_dtypes = [dim.dtype for dim in dom_dims]
 
-        if any(dim.isvar for dim in dom_dims) or \
-                dom_dims.count(dom_dims[0].dtype) != len(dom_dims):
-            return self._nonempty_domain_var()
+        extents = self._nonempty_domain_var()
```

Comment:
Would you mind to inline this so it's just a single function? I don't think it needs to be separate anymore (unless I'm missing something).

Reply:
Will do.